learning/inference.py

```diff
@@ -179,7 +179,9 @@ def main():
     )
 
     pbar_manager = enlighten.get_manager()
-    trials_pbar = pbar_manager.counter(total=args.num_trials, desc="Trials: ")
+    trials_pbar = pbar_manager.counter(
```

Comment:
I think the actions and navigation pbars can be sub-bars of trials_pbar. What do you think?

Reply:
Adding them as sub-bars could be helpful when visualizing the broader results of all of the trials rather than each individual trial. Like we could see that on average the agent completed 10% of the environment on average when navigation fills out 10% of trials progress bar. It could be confusing with actions as a sub-bar since the max actions can vary each time as we haven't decided on a standard number yet. I think all we want is as few actions as possible with highest completion. We can create a metric like completion % divided by # of actions, but I don't think actions really needs a progress bar.